### DISCUSSION TITLE
[Q&A] 如何设置 OpenAI API 接口转发 How to set up OpenAI proxy

### DISCUSSION THREAD

**Discussion by josenxx** [lobehub/lobehub#466] [Self Hosting] [answered]
please explain how to use the github repository   [vercel-proxy-openai](https://github.com/LinLin00000000/vercel-proxy-openai) to fix
<img width="584" alt="截屏2023-11-16 21 19 58" src="https://github.com/lobehub/lobe-chat/assets/10823996/8ff71c6f-86f3-440d-91c8-f574a8975d52">
this kind of problem.

**Comment by thomasnyderp** [comment, marked as answer]
You can deploy a transparent OpenAI Proxy with the button below.


[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fthomasnyderp%2Fvercel-proxy-openai)


OpenAI API url is: https://api.openai.com/v1


Then your proxy API url will be: https://xxxx.vercel.app/v1 , 

if you assign your domain. It will be: https://you-domain/v1


you can check my repo with proxy config detail: https://github.com/thomasnyderp/vercel-proxy-openai/blob/main/vercel.json

**Reply by arvinxx** [reply]
Thanks man! a clean solution!
